SU2_CFD/include/solvers/CEulerSolver.hpp

```diff
@@ -2627,6 +2643,12 @@ class CEulerSolver : public CSolver {
     ExtAverageOmega[valMarker][valSpan] = valOmega;
   }
 
+  /*!
+   * \brief Getter for TurbomachineryPerformance
+   * \return TurbomachineryPerformance container
+   */
+  inline shared_ptr<CTurbomachineryPerformance> GetTurbomachineryPerformance() const {return TurbomachineryPerformance;}
```

Comment:
This method is not really const, it returns the shared pointer by value by that pointer allows the modification of the pointed-to object. Also if the purpose here if to provide read-only access a bare const pointer would be an acceptable return type IMO (saves the reference counting etc. and should let you use unique_ptr for `TurbomachineryPerformance` instead)